imports/plugins/core/dashboard/client/templates/shop/settings/settings.js

```diff
@@ -158,12 +159,31 @@ Template.optionsShopSettings.helpers({
       shopId: Reaction.getShopId()
     });
   },
+  versionedPackages() {
+    const versionedPackages = Packages.find({ version: { $exists: true }, shopId: Reaction.getShopId() });
+    return versionedPackages;
+  },
 
   isPackageEnabled(name) {
     return Reaction.isPackageEnabled(name);
   },
 
   SitemapSettingsContainer() {
     return SitemapSettingsContainer;
+  },
+
+  PluginVersionsContainer() {
+    return PluginVersionsContainer;
+  }
+});
```

Comment:
It doesn't seem like you would need the two new functions in `optionsShopSettings` helpers. They're only used in `shopSettings` template.